src/components/EntityInfo.vue

```diff
@@ -213,7 +213,7 @@ export default class EntityInfo extends Mixins(EnumMixin) {
         href: `${sessionStorage.getItem('AUTH_URL')}business`
       },
       {
-        text: this.entityName || this.corpDisplayName,
```

Comment:
What was the difference between the entityName and the corpDisplayName?

Reply:
Previously, corpDisplayName was a method in CommonMixin that returned entityName (if not falsy), otherwise returned a string like "Numbered Benefit Company".

Now, entityTypeToTitle, which is a method in EnumMixin, just returns the string as before.

So this code is a simplification and use of a general method.